docs/beman-standard.md

```diff
@@ -0,0 +1,170 @@
+   use (commercial and non-commercial).
+3. Requires that the license appears on all copies of the software source code.
+4. Must not require that the license appears with executables or other binary
+   uses of the library.
+5. Must not require that the source code be available for execution or other
+   binary uses of the library.
+
+## General
+
+**[LIBRARY_NAMES]** REQUIREMENT: Beman libraries names begin with `Beman.`
+followed by an UpperCamelCase short name.
+
+Examples are Beman.SmartPointer and Beman.SenderReceiver.
+
+**[REPOSITORY_NAME]** RECOMMENDATION: The repository should be named after the
+library name excluding the `Beman.` prefix.
+
+For example, a Beman.SmartPointer library's repository should be named
+`SmartPointer`.
+
+## Top-level
+
+The top-level of a Beman library repository must consist of `CMakeLists.txt`,
+`LICENSE`, and `README.md` files.
+
+**[TOPLEVEL.CMAKE]** REQUIREMENT: There must be a `CMakeLists.txt` at the repository's root
+that builds and tests (via. CTest) the library.
+
+**[TOPLEVEL.LICENSE]** REQUIREMENT: There must be a `LICENSE` at the
+repository's root with the contents of an approved license that covers the
+contents of the repository.
+
+**[TOPLEVEL.README]** REQUIREMENT: There must be a markdown-formatted
+`README.md` at the repository's root that describes the library, explains how
+to build it, and links to further documentation.
+
+## `README.md`
+
+**[README.TITLE]** RECOMMENDATION: The `README.md` should begin with a level 1
+header with the name of the library optionally followed with a ":" and short
+description.
+
+For example:
+
+```markdown
+# Beman.SenderReceiver: Scalable Asychronous Program Building Blocks
+```
+
+**[README.PURPOSE]** RECOMMENDATION: Following the title, the `README.md` should
+contain a one- or two-paragraph summary describing the library's purpose.
+
+## Directory Layout
+
+**[DIRECTORY.INTERFACE_HEADERS]** REQUIREMENT: Header files that are part of the
+public interface must reside within the `include/Beman/\<ShortName\>/`
+directory.
+
+**[DIRECTORY.SOURCES]** RECOMMENDATION: Sources and headers not part of the
+public interface should reside in `src/`.
+
+## File contents
+
+**[FILE.LICENSE_ID]** REQUIREMENT: The [SPDX license
+identifier](https://spdx.dev/learn/handling-license-info/) must be added at the
+first possible line in files which can contain a comment.
+
+C++ files shall use the following form:
+
+```C++
+// SPDX-License-Identifier: <SPDX License Expression>
+```
+
+CMake files and scripts shall use the following form
+
+
+```CMake
+# SPDX-License-Identifier: <SPDX License Expression>
+```
+
+Markdown files will use a comment following the title:
+
+```markdown
+# Title
+
+<!--
+SPDX-License-Identifier: <SPDX License Expression>
+-->
+```
+
+**[FILE.COPYRIGHT]** RECOMMENDATION: Source code files should include a
+copyright notice following the SPDX license identifier.
+
+```C++
+// Copyright (c) 2024 Ron Howard <ron@boost.org>
```

Comment:
We probably don't want to use a famous actor with a non-existent email address...

Reply:
Fixed in https://github.com/beman-project/beman/pull/18/commits/d858d8eadc193713d03d3670559e260799470c37